pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -81,6 +83,25 @@ let
             ++ lib.optional (config.pulseaudio or true) libpulseaudio;
       gtk_modules = [ libcanberra-gtk2 ];
 
+      enterprisePolicies = {
+        policies = extraPolicies;
+      };
+
+      policiesJson = builtins.toFile "policies.json"
+        (builtins.toJSON enterprisePolicies);
+
+      mozillaCfg = builtins.toFile "mozilla.cfg" ''
```

Comment:
You should use `pkgs.writeText` for these, `builtins.toFile` is an evaluation time thing, which means it can't depend on any derivations, which might be wanted.

Reply:
Right, I will change that.